app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/index.tsx

```diff
@@ -164,7 +165,10 @@ const SelectColumnsView = (props: ISelectColumnsProps) => {
         });
       },
       error: (err) => {
+        dispatch({ type: 'setLoading', payload: false });
         dispatch({ type: 'setError', payload: err });
```

Comment:
are there cases we don't want to set loading to false when we set Error? If there's no such case we should just set loading to false and use 1 dispatch

Reply:
There are no other cases where we are using the `setError` dispatch (on this store).
But,I think, it's better to keep these as 2 dispatches, as these are 2 independent concerns. Currently there are no cases where we don't want to set loading to false when we set Error; but that does not mean there can not be any cases when we should not set loading to false when we set error.